conjure-macros/src/lib.rs

```diff
@@ -271,13 +271,17 @@ pub fn conjure_client(attr: TokenStream, item: TokenStream) -> TokenStream {
 ///     * `decoder` - A type implementing `DecodeParam` which will be used to decode the value.
 ///         Defaults to `FromStrDecoder`.
 ///     * `safe` - If set, the parameter will be added to the `SafeParams` response extension.
+///     * `log_as` - The name of the parameter used in request logging and error reporting. Defaults
```

Comment:
I don't love this name but can't think of a better one. I don't expect anything will use this other than conjure-codegen anyway.